modules/core/shared/src/main/scala/lucuma/core/model/Target.scala

```diff
@@ -59,14 +59,14 @@ object Target extends WithGid('t'.refined) with TargetOptics {
     val NameOrder: Order[Sidereal] = Order.by(x => (x.name, x.tracking))
   }
 
-  final case class Nonsidereal(
+  case class Nonsidereal(
```

Comment:
Is it no longer possible to extend `case class`es?

Reply:
In scala 3 you cannot extend classes unless yoou specify them with the `open` keyword
https://docs.scala-lang.org/scala3/reference/other-new-features/open-classes.html

You can still extend in particular cases (e.g. tests) but by default you can't unless explicitly enabled in the code

Reply:
In general I`ve been removing final in scala 3 code.
About `-source feature` maybe @armanbilge knows more about it?

Reply:
Btw, even though the _compiler_ enforces the `open` keyword, unless you make the class `final` it will not be `final` in the bytecode. This may have some (small) performance implications.